tests/integration_tests/data_steward/utils/bq_test.py

```diff
@@ -127,3 +131,33 @@ def test_query_sheet_linked_bq_table(self):
             expected_dict,
             columns=["site_name", "hpo_id", "site_point_of_contact"])
         pd.testing.assert_frame_equal(actual_df, expected_df)
+
+    @staticmethod
+    def get_current_gcp_account():
+        """
+        Get the account (client_email) associated with the key file
+        :return: the email of the account
+        """
+        creds_path = os.environ.get('GOOGLE_APPLICATION_CREDENTIALS')
+        with open(creds_path) as creds_fp:
+            creds = json.load(creds_fp)
+        return creds.get('client_email')
+
+    def test_update_dataset_access_entries(self):
+        dataset = self.client.create_dataset(self.dataset_id)
+        expected = dataset.access_entries.copy()
+        # add an access entry that would not be present by default
+        # valid account needed for test to pass
+        entity_id = self.get_current_gcp_account()
```

Comment:
This fails when using my application default credentials.  

1. Should there always be an email available when running this test?
2. Can we mock this instead of creating a static method to read it from the environment?